### PR TITLE
Update conf.sample.php

### PR DESCRIPTION
**Pull request by chinyasuhail**
This PR fixes a Typo in conf.sample.php

### CODE REVIEW

File: cfg/conf.sample.php [hunk 1]
```diff
@@ -33,7 +33,7 @@
 ; (optional) set a syntax highlighting theme, as found in css/prettify/
 ; syntaxhighlightingtheme = "sons-of-obsidian"
 
-; size limit per paste or comment in bytes, defaults to 10 Mebibytes
+; size limit per paste or comment in bytes, defaults to 10 Megabytes
 sizelimit = 10485760
```
Comment (rugk) [inline cfg/conf.sample.php:37]: I [double-checked the number here too](https://duckduckgo.com/?q=10+mib+in+byte&t=ffab&ia=answer) and it is the correct value for 10 MiB.

File: cfg/conf.sample.php [hunk 2]
```diff
@@ -33,7 +33,7 @@
 ; (optional) set a syntax highlighting theme, as found in css/prettify/
 ; syntaxhighlightingtheme = "sons-of-obsidian"
 
-; size limit per paste or comment in bytes, defaults to 10 Mebibytes
+; size limit per paste or comment in bytes, defaults to 10 Megabytes
```
Comment (rugk) [inline cfg/conf.sample.php:36]: I unfortunately have to say that the way it is called "Mebibytes" is correct.

These are MiB, not MB.
See https://en.wikipedia.org/wiki/Binary_prefix

**Review by rugk — COMMENTED** [review]
Hi @chinyasuhail,
thanks for your first contribution to this project! :tada: :+1: 🏅
I hope you'll like this project and enjoy hacking on it… :smiley:

However, as far as I see, this is actually not a typo.

In any way, thanks for spotting and contributing anyway, next time you'll likely spot a "real" thing… :wink:

File: cfg/conf.sample.php (hunk 1 above)
Comment (rugk) [inline cfg/conf.sample.php:37]: I [double-checked the number here too](https://duckduckgo.com/?q=10+mib+in+byte&t=ffab&ia=answer) and it is the correct value for 10 MiB.

**Comment by elrido** [comment]
Yes, sorry, in our industry we are still mostly used to the binary bytes and (to me at least) the SI-units don't line up, even though they may be more logical to non-technical users. Since the configuration file is targetted at technical users and the display logic also uses the binary prefixes, it makes sense IMO to be consistent and always use the binary bytes and not have SI-units in one place and binary bytes in the other. The other reason to stick with the binary prefixes is that they are non-ambigous: If you read MB you can never be sure if that means 1'000'000 bytes as per SI prefixes or refers to the old (up to the late 90s) practice of implicitly using SI prefixes to mean the binary ones when bytes were concerned (except for storage media sizes, where the vendors would label their hard disks i.e. 100 MB and the OS would display that as 95.4 MB).

**Pull request closed without merging**